packages/webdriverio/src/commands/browser/reloadSession.ts

```diff
@@ -39,7 +39,7 @@ export default async function reloadSession () {
         log.warn(`Suppressing error closing the session: ${err.stack}`)
     }
 
-    const ProtocolDriver = require(this.options.automationProtocol).default
+    const ProtocolDriver = require(this.options.automationProtocol!).default
```

Comment:
is that `!` intentional?

Reply:
Yes, automationProtocol is an optional value. In a different PR I will make these options required and replace usages where options is used as input with `Partial<Options>`